solutions/silver/cf-1000C.mdx

```diff
@@ -24,36 +30,29 @@ This allows us to apply prefix sums on each point in our compressed number line
 
 ```cpp
 #include <bits/stdc++.h>
-
 using namespace std;
-using ll = long long;
-using vi = vector<int>;
-using pi = pair<int, int>;
 
-#define f first
-#define s second
-#define all(x) begin(x), end(x)
+using ll = long long;
 
 int main() {
 	int n;
 	cin >> n;
 	// This stores the original endpoints to use in our frequency array later
 	vector<pair<ll, ll>> segments(n);
-	// This stores only the unique endpoints (also, the endpoints will be in
-	// sorted order).
+	// This stores only the unique endpoints in sorted order
 	set<ll> points;
 	for (int i = 0; i < n; i++) {
-		cin >> segments[i].f >>
-		    segments[i].s;  // Read in the endpoints of the i-th segment
-		// This ensures that only the unique points are stored
-		points.insert(segments[i].f);
-		points.insert(segments[i].s + 1);
+		cin >> segments[i].first >> segments[i].second;
+		// In a set, only unique points will be stored
+		points.insert(segments[i].first);
+		points.insert(segments[i].second + 1);
 	}
 
 	int cur = 0;
-	map<ll, int> compressed;  // This map stores the compressed coordinates.
+	// This map stores the compressed coordinates.
+	map<ll, int> compressed;
 	vector<ll> coords;
-	for (auto &c : points) {
+	for (const ll c : points) {
```

Comment:
maybe preserve the reference?

Reply:
![image](https://github.com/cpinitiative/usaco-guide/assets/44814054/53c2f6f5-7ec4-447e-aced-26085dbddfcf)
compiler is not happy then